tests/components/tradfri/test_cover.py

```diff
@@ -13,21 +13,15 @@
 @pytest.fixture(autouse=True, scope="module")
 def setup(request):
     """Set up patches for pytradfri methods."""
-    p_1 = patch(
+    with patch(
         "pytradfri.device.BlindControl.raw",
         new_callable=PropertyMock,
         return_value=[{"mock": "mock"}],
-    )
-    p_2 = patch("pytradfri.device.BlindControl.blinds")
-    p_1.start()
-    p_2.start()
-
-    def teardown():
-        """Remove patches for pytradfri methods."""
-        p_1.stop()
-        p_2.stop()
-
-    request.addfinalizer(teardown)
+    ):
+        with patch(
```

Comment:
We only need a single with statement and the patches can be put after it comma separated before the colon.

Reply:
I'm learning alot here, thank you!